markdown/docs/user-guide.md

```diff
@@ -514,107 +579,145 @@ export default Route.extend({
+When you run `ember build` at the command line, Ember compiles your application into the `dist` directory.
+That directory contains everything you need to make your application work in the browser.
+You can upload it to a static hosting service like S3 or Firebase, where browsers can download and run the JavaScript on the user's device.
 
-One of our goals is to one day produce a "universal" build of an Ember app
-that runs in both environments, to make release tracking easier. But for
-right now, you will need to manage two builds.
+FastBoot is a little different because, instead of being purely static, it renders HTML on the server and therefore needs more than just static hosting.
+We need to produce a build of the Ember app that's designed to work in Node.js rather than the browser.
 
-That's just what the `ember fastboot:build` command does. It
-creates a compiled version of your application with browser-specific code
-stripped out (and sometimes Node-specific code added) and puts it in the
-`fastboot-dist` directory.
+When you run `ember build`, your FastBooted app will produce differnet artifact sets in `dist/` and `dist/fastboot`.
+The artifacts in `dist` are client-side copies of your application that will be sent to all browsers that request your application.
+The artifacts in `dist/fastbot` are used by the FastBoot Server to SSR your page.
 
-You can test that the process is working (and that your app is
-FastBoot-compatible) by running `ember fastboot`, which builds your app
-then starts up a local server.
+You can test that the process is working (and that your app is FastBoot-compatible) by running `ember fastboot`, which builds your app then starts up a local server.
 
-Once you've confirmed everything looks good, it's ready to hand off to
-the FastBoot server running in the production environment. The good news
-is that this process is usually handled for you by a deployment plugin
-for `ember-cli-deploy`; see [Deploying](/docs/deploying) for more
-information about different deployment strategies.
+Once you've confirmed everything looks good, it's ready to hand off to the FastBoot server running in the production environment.
+The good news is that this process is usually handled for you by a deployment plugin for `ember-cli-deploy`; see [Deploying](/docs/deploying) for more information about different deployment strategies.
 
 ### The FastBoot Server
 
-The [FastBoot server][fastboot-server] is the heart of the FastBoot
-architecture. The server offers an [Express middleware][express] that
-can be integrated into an existing Node.js infrastructure, or run
-standalone.
+The [FastBoot server][fastboot-server] is the heart of the FastBoot architecture.
+The server offers an [Express middleware][express] that can be integrated into an existing Node.js infrastructure, or run standalone.
 
-[fastboot-server]: https://github.com/ember-fastboot/ember-fastboot-server
-[express]: http://expressjs.com
-
-```js
+```javascript
```

Comment:
Use this example? https://github.com/ember-fastboot/fastboot-app-server#quick-start